go/src/socialapi/workers/oauth/api/api.go

```diff
@@ -1,100 +0,0 @@
-package main
```

Comment:
that was a wip feature that got de-prioritized and i wanted all the things in the main repo, that is why it is in current state. On the other hand why do we build it if there is not any ref in the repo?